### PR TITLE
Project details popup

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, kindly review this PR.

### CHANGES

- Add project details popup window
- Add Javascript functionality to toggle project popup window

Thank you.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -29,6 +29,286 @@
...
+  padding: 16px;
+
+  /* gap: 24px; */
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+}
```
Comment (LionRouge1) [inline style.css]: - [x] Nice work implementing popup windows :clap: :sunglasses: However your popup design looks different compared to the Figma design for the both mobile and desktop versions (see images). Kindly, add space around the popup windows to follow the Figma design as requested in the project requirement.

### Mobile version :point_down: 
<table>
  <tr>
    <th>Your Design</th>
    <th>Figma's Design</th>
  <tr>
  <tr>
    <td></th>
    <th></th>
  <tr>
</table>

### Desktop version :point_down: 
<table>
  <tr>
    <th>Your Design</th>
    <th>Figma's Design</th>
  <tr>
  <tr>
    <td></th>
    <th></th>
  <tr>
</table>

File: script.js [hunk 2]
```diff
@@ -49,16 +32,36 @@ const card = `
...
+  mobileMenu.style.display = 'none';
+});
+
+mobileMenu.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
 
 for (let i = 0; i <= 5; i += 1) {
   cardsContainer.innerHTML += card;
 }
```
Comment (LionRouge1) [inline script.js]: - [x] This code will add the same card (project) 5 times and adding another project in the future will be a little bit challenging. Therefore I suggest you create an array of objects to store JavaScript objects containing the information for each project.

e.g: 
```sh
const projects = [
    {
        name: "project 1 name",
        description: "project 1 description",
        image: "link_to_poject1_image.jpg",
        technologies: ["HTML", "CSS", "JAVASCRIPT"],
        link: "link_to_live_version.com",
        ...
    },
    {
        name: "project 2 name",
        description: "project 2 description",
        image: "link_to_poject2_image.jpg",
        technologies: ["REACT", "CSS","REDUX"],
        link: "link_to_live_version.com",
        ...
    },
    ...]
```

File: index.html [hunk 3]
```diff
@@ -246,6 +246,78 @@ <h6>Get in touch</h6>
...
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+
+
         </div>
```
Comment (LionRouge1) [inline index.html]: - [x] Kindly make sure that the popup windows data is populated with the data from the projects on which the user clicked. That will make your popup windows dynamic.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @egichuhis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Followed Javascript best practices :+1: 
- Code clean and easy to understand :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
[OPTIONAL]
- [x] On a large screen, the bar separating the language, frameworks, and skills with their details overlaps (see image). Please check it.

Screenshot from 2023-04-18 18-50-50


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (egichuhis) [reply style.css]: Done. Thanks

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**